lua/nvim_lsp/pyls.lua

```diff
@@ -0,0 +1,28 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+
+local cwd = vim.loop.cwd()
+
+skeleton.pyls = {
+  default_config = {
+    cmd = {"pyls"};
+    filetypes = {"python"};
+    root_dir = function() return cwd end;
```

Comment:
Does pyls prefer to have one per project or one per editor?

Reply:
That's a good question -- I don't know... It's unclear from their repo. 

Any way for me to easily test this? (i.e., what should I try to see if it breaks?)

Reply:
I think it'll work fine either way, but if you do goto definition, it might consider two different directories as part of the same project. Probably fine, though. We'll change it if someone suggests otherwise.

Reply:
The only difference is setting the root dir, right? Not sure what the canonical choice would be there.